### PR TITLE
aero-check:0.1.0

### PR DESCRIPTION
**Pull request by TomVer99**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This is a template which is intended to be used to create checklists in a style that is inspired by aviation.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by TomVer99** [comment]
I have not ticked `selected a name that isn't the most obvious or canonical name for what the package does` yet, as I am not sure the name suffices. I think the name is fine, but I would like some feedback first if possible.

Thanks

File: packages/preview/aviation-style-checklist/0.1.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  }
+})
+
+#let step(a, b) = {
+  if ((a != none and a != "") or (b != none and b != "")) {
+    a; " "; box(width: 1fr,repeat[.]); " "; b; linebreak()
+  }
+}
+
+#let small_caution(
```
Comment (elegaanz) [inline packages/preview/aviation-style-checklist/0.1.0/lib.typ:58]: Public function names in Typst generally use kebab-case. If you prefer snake_case, you can also decide to keep it as is.

File: packages/preview/aviation-style-checklist/0.1.0/lib.typ [hunk 2]
```diff
@@ -0,0 +1,132 @@
+#let small_text_size = 8pt
+#let large_text_size = 14pt
+
+#let abnormals = rgb("FF9A00")
+
+#let style_state = state("style", "0")
+
+#let topic(
+  title,
+  fill_clr: rgb("FFFFFF"),
```
Comment (elegaanz) [inline packages/preview/aviation-style-checklist/0.1.0/lib.typ:10]: Same remark for function arguments.

**Review by elegaanz — COMMENTED** [review]
Looks good! Just a small remark about naming conventions, but you can ignore it if you want.
For the name, I feel like it doesn't meet the criteria, too. it would be better if you could propose something else.
Thanks :)

File: packages/preview/aviation-style-checklist/0.1.0/lib.typ (hunk 1 above)
Comment (TomVer99) [reply packages/preview/aviation-style-checklist/0.1.0/lib.typ]: Changed, no reason not to keep with the standard.

File: packages/preview/aviation-style-checklist/0.1.0/lib.typ (hunk 2 above)
Comment (TomVer99) [reply packages/preview/aviation-style-checklist/0.1.0/lib.typ]: Changed

**Comment by TomVer99** [comment]
@elegaanz would the name "AeroCheck" be good?

**Comment by elegaanz** [comment]
Yes, perfect.

**Comment by TomVer99** [comment]
Should be ready now 😄

File: packages/preview/aero-check/0.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+This template is meant to create checklists with a style inspired by aviation checklists.
+
+It includes 2 different styles!
+
+## Usage
+
+Start your checklist with the following code:
+
+```typst
+#import "@preview/aviation-style-checklist:0.1.0": *
```
Comment (elegaanz) [inline packages/preview/aero-check/0.1.0/README.md:12]: You forgot to update this import.
```suggestion
#import "@preview/aero-check:0.1.0": *
```

**Review by elegaanz — COMMENTED** [review]
Thank you for the update. Everything looks good, but an `import` in the README.

**Comment by elegaanz** [comment]
Thank you!

**Comment by TomVer99** [comment]
@elegaanz thank you for the review! And sorry for the hassle 😅 . Next time ill do it all correctly the first time.

**Pull request merged**